extensions/spatial/src/org/sbml/jsbml/ext/spatial/SpatialSpeciesPlugin.java

```diff
@@ -174,12 +174,10 @@ public boolean readAttribute(String attributeName, String prefix, String value)
 
     if (attributeName.equals(SpatialConstants.isSpatial)) {
       try {
-        setSpatial(StringTools.parseSBMLBoolean(value));
+        setSpatial(StringTools.parseSBMLBooleanStrict(value));
         isAttributeRead = true;
-      } catch (Exception e) {
-        throw new SBMLException(
```

Comment:
You should still throw the SBMLException so that a warning message is printed to the user. You should also probably keep both catch clause, your IllegalArgumentException and as well the general Exception. In both cases, you need to throw as well the SBMLException with the warning message as it was.

Reply:
Okay! I'll change this.